Cargo.toml

```diff
@@ -559,6 +559,7 @@ solana-svm = { path = "svm", version = "=2.2.0" }
 solana-svm-conformance = { path = "svm-conformance", version = "=2.2.0" }
 solana-svm-rent-collector = { path = "svm-rent-collector", version = "=2.2.0" }
 solana-svm-transaction = { path = "svm-transaction", version = "=2.2.0" }
+solana-system-interface = "=1.0.0"
```

Comment:
Let's just make this "1.0.0", since solana-system-interface is planning to use semver responsibly